### ISSUE TITLE
No module named 'third_party.DETA'

### ISSUE THREAD

**Issue by Dwrety** [janghyuncho/DECOLA#6]
While following the training guide training phase I, I found this error
`python train_net.py --num-gpus 8 --config-file DECOLA_PHASE1_Lbase_CLIP_R5021k_4x.yaml `
```
from third_party.DETA.models.deformable_detr import SetCriterion
ModuleNotFoundError: No module named 'third_party.DETA'
```
was the DETA really meant for Deformable-DETR?

**Comment by Dwrety** [comment]
I temporarily got around it by commenting everything wrt DETA in the __init__file. But got new error. 
` File "/media/Dormammu/hanz3/DECOLA/decola/modeling/backbone/timm.py", line 83, in create_timm_resnet
    default_cfgs_resnet['resnet50_in21k']['url'] = \
TypeError: 'DefaultCfg' object does not support item assignment`

**Comment by Dwrety** [comment]
Solved it by downgrade timm to 0.5.4
